package.json

```diff
@@ -19,14 +19,14 @@
     "@reach/skip-nav": "^0.17.0",
     "@react-hook/window-scroll": "^1.3.0",
     "framer-motion": "^6.5.1",
-    "next": "12.2.2",
+    "next": "12.2.3",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "styled-components": "^5.3.5",
     "swr": "^1.3.0"
   },
   "devDependencies": {
-    "@svgr/webpack": "^6.3.0",
+    "@svgr/webpack": "^6.3.1",
```

Comment:
ever since we moved the icons to its own components do we actually need this still for somethign?

Reply:
Yes, for custom SVG icons I use with SVGIcon.